apps/cli/src/commands/rollups/start.ts

```diff
@@ -1,20 +1,133 @@
 import { Command, Option } from "@commander-js/extra-typings";
 import chalk from "chalk";
 import { execa } from "execa";
+import { Listr, ListrTask } from "listr2";
+import pRetry from "p-retry";
 import path from "path";
+import { getServiceHealth } from "../../base.js";
 import { RollupsCommandOpts } from "../rollups.js";
 
 const commaSeparatedList = (value: string, _previous: string[]) =>
     value.split(",");
 
-const availableServices = [
-    "bundler",
-    "explorer",
-    "graphql",
-    // "otterscan",
-    "paymaster",
+type Service = {
+    name: string; // name of the service
+    file: string; // docker compose file name
+    healthySemaphore?: string; // service to check if the service is healthy
+    healthyTitle?: string | ((port: number) => string); // title of the service when it is healthy
+    waitTitle?: string; // title of the service when it is starting
+    errorTitle?: string; // title of the service when it is not healthy
+};
+
+const host = "http://127.0.0.1";
+
+// services configuration
+const standardServices: Service[] = [
```

Comment:
```suggestion
const baseServices: Service[] = [
```

I prefer base (since I already started using it on another PR :sweat_smile: ), but no hard opinions.